controllers/url.go

```diff
@@ -92,12 +93,33 @@ func CreateTinyURL(ctx *gin.Context, db *bun.DB) {
 		return
 	}
 
+	
 	if count >= config.UserMaxUrlCount {
 		ctx.JSON(http.StatusForbidden, dtos.URLCreationResponse{
 			Message: "You've reached the limit of " + strconv.Itoa(config.UserMaxUrlCount) + " for URLs. Delete one to add a new one !!",
 		})
 		return
 	}
+	
+	parsedUrl,err := url.Parse(body.OriginalUrl);
+
+	if err != nil {
+		return
+	}
+
+	if strings.Contains(strings.ToLower(parsedUrl.Host),strings.ToLower(config.Domain)) {
+			
+		path :=parsedUrl.Path
+	
+		if len(path) >=1  {
+
+			ctx.JSON(http.StatusForbidden, dtos.URLCreationResponse{
+				Message: "Cannot create a tiny url for tiny.realdevsquad.com",
```

Comment:
- can we use config.Domain instead of hardcoding `tiny.realdevsquad.com` to get the correct error message on staging?

Reply:
great suggestion. Working on it. Thank you!